README.md

```diff
@@ -162,10 +164,110 @@ The file extension of `content.url` defines the [tile format](#tileFormats).  Th
+
+The `transform` property applies to:
+* `tile.content`
+   * Each feature's position.
+   * Each feature's normal should be transformed by the top-left 3x3 matrix of the inverse-transpose of `transform` to account for [correct vector transforms when scale is used](http://www.realtimerendering.com/resources/RTNews/html/rtnews1a.html#art4).
+   * `content.boundingVolume`, except when `content.boundingVolume.region` is defined, which is explicitly in [WGS84 coordinates](http://earth-info.nga.mil/GandG/publications/tr8350.2/wgs84fin.pdf).
+* `tile.boundingVolume`, except when `tile.boundingVolume.region` is defined, which is explicitly in WGS84 coordinates.
+* `tile.viewerRequestVolume`, except when `tile.viewerRequestVolume.region` is defined, which is explicitly in WGS84 coordinates.
+
+The `transform` property does not apply to `geometricError`, i.e., the scale defined by `transform` does not scale the geometric error; the geometric error is always defined in meters.
+
+When `transform` is not defined, it defaults to the identity matrix:
+```json
+[
+1.0, 0.0, 0.0, 0.0,
+0.0, 1.0, 0.0, 0.0,
+0.0, 0.0, 1.0, 0.0,
+0.0, 0.0, 0.0, 1.0
+]
+```
+
+The transformation from each tile's local coordinate to the tileset's global coordinate system is computed by a top-down traversal of the tileset and post-multiplying a child's `transform` with its parent's `transform` like a traditional scene graph or node hierarchy in computer graphics.
+
+The following JavaScript code shows how to compute this using Cesium's [Matrix4](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Matrix4.js) and [Matrix3](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Matrix3.js) types.
+
+```javascript
+var stack = [];
+
+var t = rootTile; // root tile of 3D Tiles tileset
+stack.push({
+    tile : t,
+    transformToRoot : defined(t.transform) ? Matrix4.fromArray(t.transform) : Matrix4.IDENTITY;
+});
+
+while (stack.length > 0) {
+    var tt = stack.pop();
+    t = tt.tile;
+
+    var transformToRoot = tt.transformToRoot;
+    // Apply 4x4 transformToRoot to this tile's positions and bounding volumes
+
+    var inverseTransform = Matrix4.inverse(transformToRoot, new Matrix4());
+    var normalTransform = Matrix4.getRotation(inverseTransform, new Matrix3());
+    normalTransform = Matrix3.transpose(normalTransform, normalTransform);
+    // Apply 3x3 normalTransform to this tile's normals
+
+    var children = t.children;
+    var length = children.length;
+    for (var k = 0; k < length; ++k) {
+        var child = children[k];
+        var childToRoot = defined(child.transform) ? Matrix4.fromArray(child.transform) : Matrix4.IDENTITY;
+        childToRoot = Matrix4.multiplyTransformation(transformToRoot, childToRoot, childToRoot);
+        stack.push({
+            tile : child,
+            transformToRoot : childToRoot
+        });        
+    }
+}
+```
```

Comment:
Ah, yes, it is much shorter.  If you have a Cesium unit test handy, could you quickly run this?